pubs/content.py

```diff
@@ -52,16 +52,17 @@ def _open(path, mode):
     else:
```

Comment:
No no no. To the title of you commit. But I did not find any way to comment that directly.